Assets/Scripts/playertwo.cs

```diff
@@ -4,38 +4,40 @@
 
 public class playertwo : MonoBehaviour
```

Comment:
like discussed yesterday. If the two scripts (playerone and playertwo) are equals, you can remove one of both and assign the single script to both players